.env

```diff
@@ -12,7 +12,7 @@ OFF_GID=1000
 # Robotoff
 TAG=latest
 ROBOTOFF_INSTANCE=localhost
-ROBOTOFF_DOMAIN=openfoodfacts.localhost
+ROBOTOFF_DOMAIN=openfoodfacts.net
```

Comment:
This is _only_ used (AFAIU) to download a JSON file from PO.

I'm not very happy about it - what alternatives do we have?

Reply:
Not only for JSON - look at the usage of `BaseURLProvider`.

What are you not happy about in particular, the file downloads/the fact we have to manipulate URLs to get things working?

Reply:
Hm, maybe I should have clarified - I meant in the integration tests. But it might not be true even there of course :)

What I'm unhappy with is how the test isn't hermetic - if the .net environment is down the tests will fail.

Reply:
We can construct a test taxonomy JSON with the one category we need in tests and mock out the actual call, returning the test taxonomy instead?

Reply:
Actually, seems we can just use a [similar set up to this test](https://github.com/openfoodfacts/robotoff/blob/837c2da66b80e48a47788dc92d7975bd6c569424/tests/test_taxonomy.py#L9), since Robotoff already has a fallback taxonomy JSON stored

Reply:
That's a promising idea, but the test you linked doesn't make the API in general use a different source for JSON files, it just validates that using those local files works, right?

I'm looking at how to tell the entire Robotoff engine to use the local files instead..

Reply:
That's a promising idea, but the test you linked doesn't make the API in general use a different source for JSON files, it just validates that using those local files works, right?

I'm looking at how to tell the entire Robotoff engine to use the local files instead..

Reply:
Alright, when I started looking (and trying to make the taxonomy JSON specifically load locally) I noticed that Robotoff uses PO to load product images and things as well.

I think it will have to rely on the .net environment until we get enough problems from it that it's worth either mocking up a productopener, or forcing the integration tests to run a local productopener as well.